strax/context.py

```diff
@@ -685,21 +688,31 @@ def check_cache(d):
                 chunk_number=chunk_number,
                 time_range=time_range)
 
-            if not ldr and run_id.startswith('_'):
+            _is_superrun = run_id.startswith('_') and not p.provides[0].startswith('_temp')
+            if not ldr and _is_superrun:
                 if time_range is not None:
                     raise NotImplementedError("time range loading not yet "
                                               "supported for superruns")
 
                 sub_run_spec = self.run_metadata(
                     run_id, 'sub_run_spec')['sub_run_spec']
+
+
+                # Make subruns if they do not exist, since we do not 
+                # want to store data twice in case we store the superrun
+                # we have to deactivate the storage converter mode.
+                stc_mode = self.context_config['storage_converter']
+                self.context_config['storage_converter'] = False
                 self.make(list(sub_run_spec.keys()), d)
+                self.context_config['storage_converter'] = stc_mode
```

Comment:
Depends, these lines are actually coming from a previous version in which I had not added yet `write_superruns` as context config. I thought it made sense to keep it in case we ether want to copy only the superrun from one storage frontend to another but not its subruns.

Reply:
But I also do not mind to change it. I am not even sure if this will be ever a use case.